shub/deploy.py

```diff
@@ -140,17 +141,17 @@ def _upload_egg(target, eggpath, project, version, auth):
     return make_deploy_request(url, data, files, auth)
 
 
-def _build_egg():
+def _build_egg(setup_file):
     closest = closest_scrapy_cfg()
     os.chdir(os.path.dirname(closest))
-    if not os.path.exists('setup.py'):
+    if not os.path.exists(setup_file):
```

Comment:
@bbotella from my experience users (myself included) don't pay too much attention to those msgs as long as there's a success in the end.

@eliasdorneles That's another option that I've considered. I checked the internal project and found out that we've been using two setup files for quite some time, so the use case seemed to be legitimate.